pkgs/misc/drivers/em100/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, stdenv
+, curl
+, pkgconfig
+, libusb
+, fetchgit
+}:
+stdenv.mkDerivation rec {
+  pname = "em100";
+  version = "unstable-2022may02-86631c9";   # upstream does not appear to have formal releases
+  src = fetchgit {
+    url  = "https://review.coreboot.org/em100";
+    rev  = "7cd8c935239ab63ea7dd583183d8144e38c42186";
+    hash = "sha256-zKRVhqNHtgVj6JDkTV+6GEjmM1FICqQr286uTXwUvpo=";
+  };
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [
+    pkgconfig
+  ];
+
+  buildInputs = [
+    libusb
+    curl    # for optionally downloading firmware images
+  ];
+
+  installPhase = ''
```

Comment:
When overriding a phase remember to add the `runHooks for its pre and post phases`